components/gitpod-db/src/typeorm/migration/1659601647550-BilledSession.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class BilledSession1659601647550 implements MigrationInterface {
```

Comment:
How did you generate this migration? My own process for generating a migration for a new table using typeorm is somewhat convoluted 😬 

I think there is an opportunity to improve the [README](https://github.com/gitpod-io/gitpod/blob/6c4e272bbcb330ff3259fb51767f307387f7213a/components/gitpod-db/README.md) here.

Reply:
Ohh, it's this [README](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-db/src/typeorm/migration/README.md) instead of the one you linked above. Definitely an opportunity to improve how we point to the right docs 🙈